validator/webui/webui.js

```diff
@@ -46,3 +48,31 @@ function setLocationHashParams(params) {
   }
   window.location.hash = out.join('&');
 }
+
+// Base64 encoded ascii to ucs-2 string.
+function atou(str) {
+  return decodeURIComponent(escape(atob(str)));
+}
+
+// Extract queryParams from url and return object.
+function getSearchParams() {
+  const {search} = window.location;
```

Comment:
shall we use fragment? since we have `#htmlFormat=AMP4ADS`

Reply:
discussed offline, one other reason to use fragment is so that no data is transfer to the server. that in general prevent any potential privacy related issue (PII in the data etc).

Reply:
Done.